interface/modules/custom_modules/oe-module-claimrev-connect/templates/eligibility.php

```diff
@@ -112,133 +112,139 @@
                                     <div class="row">
                                         <div class="col">
-            <?php
+                <?php
                                 include $path . '/deductibles.php';
-            ?>
+                ?>
                                         </div>
                                     </div>
                                 </div>
-                                <div id="eligibility-medicare" class="tab-pane">
+                                <div id="eligibility-medicare-<?php echo($index); ?>" class="tab-pane">
                                     <div class="row">
                                         <div class="col">
-            <?php
+                <?php
                                 include $path . '/medicare_info.php';
-            ?>
+                ?>
                                         </div>
                                     </div>
                                 </div>
-                                <div id="eligibility-benefits" class="tab-pane">
+                                <div id="eligibility-benefits-<?php echo($index); ?>" class="tab-pane">
                                  
                                     <div class="row">
                                         <div class="col">
-            <?php
+                <?php
                                 $source = $data->informationSourceName;
                                 include $path . '/source.php';
-            ?>
+                ?>
                                         </div>
                                     </div>
                                     <div class="row">
                                         <div class="col">
-            <?php
+                <?php
                                 $receiver = $data->receiver;
                                 include $path . '/receiver.php';
-            ?>
+                ?>
                                         </div>
                                     </div>
 
                                     <div class="row">
                                         <div class="col">
-            <?php
-            if ($benefits != null) {
-                        include $path . '/subscriber_patient.php';
-                        include $path . '/benefit.php';
-            }
-            ?>
+                    <?php
+                    if ($benefits != null) {
+                                include $path . '/subscriber_patient.php';
+                                include $path . '/benefit.php';
+                    }
+                    ?>
                                         </div>
                                     </div>
                                 </div>
-                                <div id="eligibility-validations" class="tab-pane">
+                                <div id="eligibility-validations-<?php echo($index); ?>" class="tab-pane">
```

Comment:
to be safe would html escape all the $index with attr function:
```
<?php echo(attr($index)); ?>
```

Reply:
done